lib/liveman/user/schemas/user_schema.ex

```diff
@@ -3,15 +3,49 @@ defmodule Liveman.User.Schemas.User do
 
   import Ecto.Changeset
 
+  @email_regex_format ~r/^\S+@\S+\.\S+$/
+
+  @derive {Inspect, except: [:password]}
```

Comment:
Why is this needed? I am curious as I have not used it and did not see it before 😅

Reply:
@olivierobert This one is for hide sensitive data from inspect, log etc. It will not show the `password`.  https://hexdocs.pm/elixir/Inspect.html#module-deriving
<img width="1162" alt="Screen Shot 2021-07-30 at 2 50 16 PM" src="https://user-images.githubusercontent.com/6076762/127620049-8957c943-5c2c-4a66-b59a-11d946209e7a.png">